include/oneapi/dpl/internal/inclusive_scan_by_segment_impl.h

```diff
@@ -20,7 +20,7 @@
 #define _ONEDPL_INCLUSIVE_SCAN_BY_SEGMENT_IMPL_H
 
 #include "by_segment_extension_defs.h"
-#include "../pstl/glue_numeric_impl.h"
+#include "../numeric" // inclusive_scan, transform_inclusive_scan
```

Comment:
I think it is better way just to include `glue_numeric_defs.h` for making the necessary declarations visible.

Reply:
Done